AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -1060,7 +1060,7 @@ vstup.osvita.ua##.section-subscription
 news24.com##.article-subs-footer
 voelkner.de##.footer_top__content_container
 cnn.co.jp##.line-news-banner
-newdeal.by,yetkinreport.com##.mailpoet_form_popup_overlay
+trendblog.euronics.de,newdeal.by,yetkinreport.com##.mailpoet_form_popup_overlay
```

Comment:
I see it in the code, but it does not appear on my end.
Is page scrolling blocked?

Reply:
Nvm. I got this.